analyzers/tests/SonarAnalyzer.UnitTest/Rules/ArrayDesignatorOnVariableTest.cs

```diff
@@ -25,15 +25,16 @@ namespace SonarAnalyzer.UnitTest.Rules
     [TestClass]
     public class ArrayDesignatorOnVariableTest
     {
+        private readonly VerifierBuilder builder = new VerifierBuilder<ArrayDesignatorOnVariable>().AddPaths("ArrayDesignatorOnVariable.vb");
```

Comment:
This one is very tricky. It took me a few moments to understand why the code fix works without an initial path :) (which means I actually missed that the path was missing at line 32). I would move the paths to the test since like this is confusing and we will have to anyway when we will add more tests to this class.